released/packages/coq-iris-string-ident/coq-iris-string-ident.0.1.0/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+name: "coq-iris-string-ident"
+maintainer: "Ralf Jung <jung@mpi-sws.org>"
+authors: "Tej Chajed <tchajed@mit.edu>"
+license: "BSD"
+homepage: "https://gitlab.mpi-sws.org/iris/string-ident"
+bug-reports: "https://gitlab.mpi-sws.org/iris/string-ident/issues"
+dev-repo: "git+https://gitlab.mpi-sws.org/iris/string-ident.git"
+
+synopsis: "Add support for Gallina names in intro patterns to the Iris Proof Mode"
+description: """
+This package implements string_to_ident in Ltac2, enabling support for Gallina names in Iris intro patterns (within strings).
+"""
+
+depends: [
+  "coq" {>= "8.11"}
```

Comment:
Should probably add the `& coq < "8.13"` that I also added on our side as we cannot test this against Coq 8.3.

Reply:
Either is fine by me. If for some reason the package stops to work with, say, 8.13.0, Guillaume Claret's coq-bench will find this, and we will disable the package for `< "8.13"`.